lib/taskana-core-test/src/test/java/acceptance/taskcomment/delete/DeleteTaskCommentAccTest.java

```diff
@@ -0,0 +1,161 @@
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+    comment1 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    comment2 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    List<TaskComment> taskComments = taskService.getTaskComments(task1.getId());
+    assertThat(taskComments).hasSize(2);
+  }
+
+  @WithAccessId(user = "admin")
+  @AfterEach
+  public void deleteTask() throws Exception {
+    taskService.cancelTask(task1.getId());
+    taskService.deleteTask(task1.getId());
+  }
+
+  @WithAccessId(user = "admin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassification =
+        DefaultTestEntities.defaultTestClassification().buildAndStore(classificationService);
+    defaultWorkbasket =
+        DefaultTestEntities.defaultTestWorkbasket().buildAndStore(workbasketService);
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasket.getId())
+        .accessId(ACCESS_USER_1_1_ID)
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+  }
+
+  @WithAccessId(user = ACCESS_USER_1_1_ID)
+  @Test
+  void should_DeleteTaskComment_For_TaskCommentId() throws Exception {
+
+    taskService.deleteTaskComment(comment1.getId());
+
+    List<TaskComment> taskCommentsAfterDeletion = taskService.getTaskComments(task1.getId());
+    assertThat(taskCommentsAfterDeletion).hasSize(1);
+  }
+
+  @WithAccessId(user = "user-1-2", groups = "user-1-1") // to read comments
+  @Test
+  void should_FailToDeleteTaskComment_When_UserHasNoAuthorization() throws Exception {
+    assertThatThrownBy(() -> taskService.deleteTaskComment(comment1.getId()))
+        .isInstanceOf(NotAuthorizedException.class);
+
+    List<TaskComment> taskCommentsAfterDeletion = taskService.getTaskComments(task1.getId());
+    assertThat(taskCommentsAfterDeletion).hasSize(2);
+  }
+
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_DeleteTaskComment_When_UserIsInAdministrativeRole() throws Exception {
+    taskService.deleteTaskComment(comment1.getId());
+
+    List<TaskComment> taskCommentsAfterDeletion = taskService.getTaskComments(task1.getId());
+    assertThat(taskCommentsAfterDeletion).hasSize(1);
+  }
+
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_DeleteTaskComment_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
```

Comment:
This test seems similar to the should_DeleteTaskComment_When_UserIsInAdministrativeRole